scripts/missions/windurst/1_3_The_Price_of_Peace.lua

```diff
@@ -0,0 +1,325 @@
+            ['Tih_Pikeh']   = mission:event(152),
+        },
+    },
+
+    -- Speak to Leepe-Hoppe on the roof of the Rhinostery near Waters HP #3
+    {
+        check = function(player, currentMission, missionStatus)
+            return currentMission == mission.missionId and missionStatus == 1
+        end,
+
+        [xi.zone.WINDURST_WATERS] =
+        {
+            ['Kenapa-Keppa']   = mission:event(145),
+            ['Leepe-Hoppe']    = mission:progressEvent(140),
+            ['Ohbiru-Dohbiru'] = mission:event(143),
+
+            onEventFinish =
+            {
+                [140] = function(player, csid, option, npc)
+                    npcUtil.giveKeyItem(player, {
+                        xi.ki.FOOD_OFFERING,
+                        xi.ki.DRINK_OFFERING
+                    })
+                    player:setMissionStatus(mission.areaId, 2)
+                end,
+            },
+        },
+    },
+
+    -- Take the Offerings to Giddeus and give them to the Yagudo NPCs
+    -- On retail, the Yagudo NPCs repeat the same dialogue until after both
+    -- offerings are turned in, even on repeat interactions
+    {
+        check = function(player, currentMission, missionStatus)
+            return currentMission == mission.missionId and missionStatus == 2
+        end,
+
+        [xi.zone.WINDURST_WATERS] =
+        {
+            ['Leepe-Hoppe'] = mission:progressEvent(142),
+        },
+
+        [xi.zone.GIDDEUS] =
+        {
+            ['Ghoo_Pakya'] = mission:progressEvent(49),
+            ['Laa_Mozi']   = mission:progressEvent(45),
+
+            onEventFinish =
+            {
+                [45] = offeringsTurnedIn,
+                [49] = offeringsTurnedIn,
+            },
+        },
+    },
+
+    -- Offerings were delivered
+    {
+        check = function(player, currentMission, missionStatus)
+            return currentMission == mission.missionId and missionStatus == 3
+        end,
+
+        [xi.zone.GIDDEUS] =
+        {
+            ['Ghoo_Pakya'] = mission:event(52):replaceDefault(),
+            ['Laa_Mozi']   = mission:event(48):replaceDefault(),
+        },
+
+        [xi.zone.WINDURST_WATERS] =
+        {
+            ['Ohbiru-Dohbiru'] = mission:event(144),
+
+            onRegionEnter =
+            {
+                [1] = function(player, region)
+                    return mission:progressEvent(146)
+                end,
+            },
+
+            onEventFinish =
+            {
+                [146] = function(player, csid, option, npc)
+                    player:setMissionStatus(mission.areaId, 4)
+                end,
+            },
+        },
+    },
+
+    -- Finish mission with a gate guard
+    {
+        check = function(player, currentMission, missionStatus)
+            return currentMission == mission.missionId and missionStatus == 4
+        end,
+
+        [xi.zone.PORT_WINDURST] =
+        {
+            ['Janshura-Rashura'] =  mission:progressEvent(114),
```

Comment:
Theres a bunch of funky styling around here, you can find it by searching for "=  " (equals space space)